src/contracts/interfaces/IStrategyManager.sol

```diff
@@ -60,12 +60,12 @@ interface IStrategyManager {
 
     /**
      * @notice Records an overcommitment event on behalf of a staker. The staker's beaconChainETH shares are decremented by `amount`.
-     * @param overcommittedPodOwner is the pod owner to be slashed
+     * @param podOwner is the pod owner to be slashed
      * @param beaconChainETHStrategyIndex is the index of the beaconChainETHStrategy in case it must be removed,
-     * @param amount is the amount to decrement the slashedAddress's beaconChainETHStrategy shares
+     * @param sharesDelta is the change in podOwner's beaconChainETHStrategy shares
      * @dev Only callable by EigenPodManager.
```

Comment:
this documentation needs updating too (just write it for the contract and then copy-paste into interface, I guess)

Reply:
fixed